test/deploy/rackhd_source_install.py

```diff
@@ -108,7 +94,7 @@ def test02_clone_rackhd_source(self):
                                                  + fit_common.fitinstall()['rackhd']['branch']
                                                  )['exitcode'], 0, "Branch not found on RackHD repo.")
         # clone modules
-        for repo in modules:
+        for repo in fit_common.fitinstall().keys():
```

Comment:
Okay, if this is something you're still planning to do.   Is there an advantage or that deployment vs. the current playbook?